cfgov/v1/models/organisms.py

```diff
@@ -106,6 +106,16 @@ class Meta:
         template = '_includes/organisms/main-contact-info.html'
 
 
+class SidebarContactInfo(blocks.StructBlock):
+    header = blocks.CharBlock(required=False)
```

Comment:
`content.header` is not wrapped in an `if` statement in `sidebar-contact-info.html`, should it be if this is not required?